spec/models/user_spec.rb

```diff
@@ -35,4 +35,18 @@
       expect(arthur.username).to eq("arthur")
     end
   end
+
+  describe "#collection method" do
+    let(:john) { create(:user) }
+    before do
+      5.times do
```

Comment:
2 would be enough, right?

Reply:
Haha yeah you are right. Ok i'll reduce the test data